dev_guide/templates.adoc

```diff
@@ -97,15 +118,15 @@ by piping both commands:
 $ oc process -f <filename.json> | oc create -f -
 ----
 
-You can override any link:../dev_guide/templates.html#parameters[parameters]
-defined in the JSON file by adding the `-v` option followed by a
+You can override any link:../dev_guide/templates.html#parameters[parameter]
+values defined in the JSON file by adding the `-v` option followed by a
 comma-separated list of _<name>_=_<value>_ pairs.
 A parameter reference may appear in any text field inside the template items.
 
 For example, you can override the *`ADMIN_USERNAME`* and *`MYSQL_DATABASE`*
 parameters to create a configuration with customized environment variables:
 
-.Generating a Configuration
+.Generating a list of resources
```

Comment:
this isn't generating a list of resources, it's creating resources from a template.